docs/receipts.md

```diff
@@ -15,14 +15,70 @@ In the _Document Templates_ section of the Indico administration page:
 
 Indico lets you define templates for various types of documents, including receipts, certificates, and more. These templates allow you to specify the layout, content, and placeholders for dynamic data using HTML with Jinja2 templates.
 
-Templates can be either created under an event, or a category, making them available for use in all of its events. These can be viewed, added, cloned and deleted from the management area under the _Customisation_ tab on events, or on its own _Document Templates_ tab on categories.
+Templates can be either created under an event, or a category, making them available for use in all of its events. These can be viewed, added, cloned and deleted from the management area under the _Customisation_ tab in events, or in its own _Document Templates_ tab on categories.
 
 ![](assets/document_templates/edit.png)
 
-When creating a new template you must specify its name, and optionally define a default filename. Then you can design your template using the code editor. You can write HTML with Jinja2 templates, CSS and specify custom variables in YAML.
+When creating a new template you must specify its name, and optionally define a default filename. Then you can design your template using the code editor. You can write HTML with Jinja2 templates, CSS and specify custom fields in YAML.
 
 ![](assets/document_templates/create.png)
 
+#### Defining custom fields
+
+The custom variables are specified by defining a `custom_fields` array. For each element, you can set the following keys:
```

Comment:
I think it might not be clear where you should add the custom variables:
```suggestion
The custom variables are specified by defining a `custom_fields` array in the `metadata.yml` file. For each element, you can set the following keys:
```